test/InterchainToken.js

```diff
@@ -147,7 +147,7 @@ describe('InterchainToken', () => {
             const contractBytecodeHash = keccak256(contractBytecode);
 
             const expected = {
-                london: '0xa01cf28b0b6ce6dc3b466e995585d69486400d671fce0ea8d06beba583e6f3bb',
+                london: '0x75a10f371014f97a894e69395422a36cc9e9961e32371c0213ed514702966b24',
```

Comment:
Why is this changing? These bytecode checks exist to catch unexpected modifications

Reply:
I flattened the `InterchainToken.sol` file and looked into the differences. I found that some functions were updated by this PR: https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/pull/189.

i.e.: _toAccountRoles
    `function _toAccountRoles(uint8[] memory roles) internal pure returns (uint256)  `
vs
   ` function _toAccountRoles(uint8[] memory roles) internal pure returns (uint256 accountRoles)`

Let me know if we're okay with this change or not.

Reply:
i see, let's revert it, and release 6.0.1

Reply:
Merged PR: https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/pull/194 and release snapshot, and now the bytecode doesn't needs to be updated.